cluster/gce/gci/configure-helper.sh

```diff
@@ -1927,6 +1927,35 @@ function setup-etcd-encryption {
   fi
 }
 
+# Updates node labels used by addons.
+function update-legacy-addon-node-labels() {
```

Comment:
nit: probably add a log line for this step?

Reply:
To make sure I understand it correctly --- this step is needed for the upgrade case, in which kubelet won't be able to update existing node to have the new label even if it is given via `--node-labels`?

Reply:
> this step is needed for the upgrade case, in which kubelet won't be able to update existing node to have the new label even if it is given via `--node-labels`?

exactly